### ISSUE TITLE
Fix false positives for prefers-reduced-motion media in media-feature-name-no-unknown

### ISSUE THREAD

**Issue by pkuczynski** [stylelint/stylelint#3396]
> Clearly describe the bug

`media-feature-name-no-unknown` reports an error when `prefers-reduced-motion` is used. This conflicts with this rule: https://github.com/YozhikM/stylelint-a11y/blob/master/src/rules/media-prefers-reduced-motion/README.md

> Which rule, if any, is the bug related to?

`media-feature-name-no-unknown`

> What CSS is needed to reproduce the bug?

e.g.

```css
@media screen and (prefers-reduced-motion) {
  div {
    animation: none;
  }
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "'media-feature-name-no-unknown": "true"
  }
}
```

> Which version of stylelint are you using?

`9.2.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`stylelint --syntax scss  './src/**/*.scss'`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No warnings to be flagged

> What actually happened (e.g. what warnings or errors did you get)?

"The following warnings were flagged:"

```shell
test.scss
 19:20  ✖  Unexpected unknown media feature name "prefers-reduced-motion"   media-feature-name-no-unknown
```

**Comment by jeddy3** [comment]
@pkuczynski Thanks for the report and for using the template.

I've labelled this up as a bug.  As the issue has also been labelled "help wanted", please consider [contributing a fix](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule). It should just be a case of adding the media feature name to [the reference list](https://github.com/stylelint/stylelint/blob/75172b66dfa50bc84775315accf039a305c4c7f7/lib/reference/keywordSets.js#L553-L590), (and an test).

**Comment by pkuczynski** [comment]
@jeddy3 I already did. Check #3397
